### ISSUE TITLE
Redot Engine's update checker grabs new update from Godot instead

### ISSUE THREAD

**Issue by MudkipWithShades** [Redot-Engine/redot-engine#808]
### Tested versions

- Reproducible in: v4.3.beta.1.official [7e93ceddf]

### System information

Redot v4.3.beta.1 - Windows 10.0.19045 - GLES3 (Compatibility) - NVIDIA GeForce RTX 3060 Ti (NVIDIA; 32.0.15.6603) - AMD Ryzen 5 5600X 6-Core Processor (12 Threads)

### Issue description

When Redot says there is a new update, it takes you to the Godot website to down a new version, but that is not Redot. It should download a new update whenever Redot has a new version available, not Godot's.
![issue](https://github.com/user-attachments/assets/331760d9-33b4-491b-b7c8-a95432789357)

### Steps to reproduce

- Go into Settings, set Network Mode to "Online"
- Close Redot
- Click on Update Available, it takes you straight to Godot's website instead of Redot's.

### Minimal reproduction project (MRP)

This is on the Project Menu whenever Redot is started.

**Comment by TimeVeteran** [comment]
Yeah, found this problem as well. I do have a question though, will the next version be named 4.4 or 4.5? This is confusing because there is **already** a 4.4 version.

**Comment by MudkipWithShades** [comment]
> Yeah, found this problem as well. I do have a question though, will the next version be named 4.4 or 4.5? This is confusing because there is **already** a 4.4 version.

As far as I know, the next version will just be named 4.4. I don't think the Godot team has started working on 4.5 yet.

**Comment by TimeVeteran** [comment]
I'm talking about Redot though, and we already have a 4.4 alpha that was released earlier.

**Comment by mcdubhghlas** [comment]
Heads up: I have fixed this issue, I want to wait for the web dev side of things before I can push it through.

**Comment by Spartan322** [comment]
This has been fixed for the next 4.3 beta

@TimeVeteran 

> Yeah, found this problem as well. I do have a question though, will the next version be named 4.4 or 4.5? This is confusing because there is already a 4.4 version.

We are following Godot's major and minor versions, our 4.4 will be compatible with Godot's 4.4, our 4.3 will be compatible with Godot's 4.3, 4.4 and 4.3 are different versions, 4.3 is the more stable version, 4.4 is the bleeding edge, carrying more features but is less stable and robust.

**Comment by TimeVeteran** [comment]
Alright, I understand it more now.

**Comment by Spartan322** [comment]
> This is still replicated using v.4.3.rc.1.official

That's not the same issue, its related, but this bug specifically refers to Redot retrieving update data from Godot.

**Comment by Arctis-Fireblight** [comment]
Looks like this was fixed long ago and never closed. Closing this now.
